.circleci/src/workflows/commit.yml

```diff
@@ -5,39 +5,108 @@ jobs:
         - mattermost
       delivery: none
 
+  - build_web:
+      name: build_ig_test_cms_web
+      build_config_name: integreat-test-cms
+      context:
+        - mattermost
+      requires:
+        - bump_version
+
+  # On feature branches: Execute checks and e2e tests
+
   - check:
+      filters:
+        branches:
+          ignore:
+            - main
       context:
         - mattermost
   - e2e_web:
+      filters:
+        branches:
+          ignore:
+            - main
       context:
         - mattermost
       requires:
         - bump_version
         - check
-  - build_web:
-      name: build_ig_test_cms_web
-      build_config_name: integreat-test-cms
+
+  - build_android:
+      filters:
+        branches:
+          ignore:
+            - main
+      name: build_e2e_android
+      build_config_name: integreat-e2e
       context:
         - mattermost
+        - tuerantuer-google-play
+        - tuerantuer-android
+      requires:
+        - bump_version
+        - check
```

Comment:
missing newline

Reply:
Not missing but by purpose to group jobs that belong together.